### ISSUE TITLE
TV4Play - playback issue on some of the episodes

### ISSUE THREAD

**Issue by Sopor** [spaam/svtplay-dl#982]
It seems that the video from TV4Play doesn't play smoothly. Most of the time it plays as it should but suddenly it starts to lag a few seconds and i can see how the picture is jerky (jag tar det på svenska, det hackar, rycker i bilden och släpar efter) and then it is back to normal again and after a few minutes it happens again. I haven't seen this behavior on any other play site.
If i watch on TV4Play or if i use Retrospect it plays without this issue.
It is not every episode that have this issue. You can compare these two episodes:
`https://www.tv4play.se/program/biggest-loser-vip/3971462` with
`https://www.tv4play.se/program/biggest-loser-vip/3971779` and see if you notice any difference between them. I have downloaded them more than one time so it wasn't a temporary issue with the download but it will be the same.

I believe it was @JohnnyJorgensen that also experiences that it was not running smoothly.

**Comment by JohnnyJorgensen** [comment]
@Sopor- Yes, but my synchronization problem disappered when I used the special ffmpeg version some other user was referring to in the other thread... Haven't tried with the above two links, though...

**Comment by spaam** [comment]
hmm.. @JohnnyJorgensen  what version is that? 
@Sopor- i have no idea to fix it.  feels its outside of the script

**Comment by JohnnyJorgensen** [comment]
It was the one Bengt-H talked about here:
https://github.com/spaam/svtplay-dl/issues/925#issuecomment-407211537

**Comment by Sopor-** [comment]
It is the same for `https://www.tv4play.se/program/brottsjournalen`. The picture is really bad and i can see how it lags. This must be something that made this when we remux or merge?

**Comment by Sopor-** [comment]
@spaam What command switches do you use for ffmpeg when merging audio and video?

**Comment by Sopor-** [comment]
Latest Parlamentet is out of sync too :(
